api/v1alpha1/oadp_types.go

```diff
@@ -177,10 +177,8 @@ type ResticConfig struct {
 // ApplicationConfig defines the configuration for the Data Protection Application
 type ApplicationConfig struct {
 	Velero *VeleroConfig `json:"velero,omitempty"`
-	// (deprecation warning) ResticConfig is the configuration for restic DaemonSet.
-	// restic is for backwards compatibility and is replaced by the nodeAgent
-	// restic will be removed in the future
-	// +kubebuilder:deprecatedversion:warning=1.3
+	// (do not use warning) restic field is for backwards compatibility and
```

Comment:
What does (do not use warning) mean here ?

Reply:
I think the intend here should be that "If a user uses restic config then the operator should do a No-Op and should add a warning on DPA that use node-agent config instead of restic"

Reply:
So, the current idea is that user can only use restic through node-agent config, this is a way to tell user not to use restic config

If user tries to use, DPA want reconcile telling user to use node-agent config instead